### ISSUE TITLE
CCK Black Box Study is appearing in Rosetta

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#151]
While working on #150 I noticed that there are some translations for Circuit Construction Kit: Black Box Study.  I asked @samreid about it via Skype, and he said it shouldn't be there.  Here is the dialog:

[1:38:43 PM] John Blanco: Yo Sam - I'm doing some translation work, and I noticed some translations for cckbbs.  I just checked in Rosetta and it is indeed appearing.  Is this desired or not?
[1:39:01 PM] Sam Reid: It is unexpected
[1:39:17 PM] Sam Reid: One sad day, CCKBBS was accidentally published as “phet” instead of “phet-io”
[1:39:28 PM] Sam Reid: We thought it wouldn’t be a problem since it was never made public with a sim page.
[1:39:48 PM] Sam Reid: however, JO and MP recently had to strip its entries out of database so it wouldn’t interfere with something.
[1:39:55 PM] Sam Reid: Sorry it is causing trouble for rosetta.
[1:39:58 PM] Sam Reid: Not sure what to do
[1:40:19 PM] Sam Reid: It seems we should tell rosetta that it doesn’t exist.
[1:41:22 PM] John Blanco: I'll get it taken care of.  Just thought I'd verify with you that it shouldn't be there.

**Comment by jbphet** [comment]
This sim was appearing in Rosetta because the sim info had made it into a file in the Rosetta repo named `SimInfoArray.json`.  This file is updated by the build server when a new sim is added, but it doesn't appear that there is any mechanism in place to take sims out of it, so I just manually removed CCKBB.  I pulled on the server and checked Rosetta, and the sim did indeed disappear from the list of translatable sims.  I'll put a reminder on my calendar to check in a few days after we've had a few translations submitted that this doesn't reappear.
